ci-operator/config/openshift/cincinnati/openshift-cincinnati-master__stablerust.yaml

```diff
@@ -0,0 +1,141 @@
+base_images:
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+  rust:
+    name: rust
+    namespace: ci
+    tag: 1.47.0
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_literal: |
+    FROM rust
+    COPY . /opt/app-root/src/
+    WORKDIR /opt/app-root/src/cincinnati
+    USER 0
+    RUN chgrp root /opt/app-root/src/cincinnati && \
```

Comment:
Required for `cargo-test` to create binaries (it runs unprivileged with `root` group)